README.md

```diff
@@ -228,6 +230,88 @@ then `created_at` column is set from time attribute in a fluentd packet with tim
 +-----+-----------+---------------------+
 ```
 
+## Configuration Example(bulk insert with tag placeholder for table name)
+
+```
+<match mysql.input>
+  @type mysql_bulk
+  host localhost
+  database test_app_development
+  username root
+  password hogehoge
+  column_names id,user_name,created_at
+  key_names id,user,${time}
+  table users_${tag}
+  <buffer tag>
+    @type memory
+    flush_interval 60s
+  </buffer>
+</match>
+```
+
+Assume following input is coming:
+
+```js
+2016-09-26 18:42:13+09:00: mysql.input: {"user":"toyama","dummy":"hogehoge"}
+2016-09-26 18:42:16+09:00: mysql.input: {"user":"toyama2","dummy":"hogehoge"}
+2016-09-26 18:42:19+09:00: mysql.input: {"user":"toyama3","dummy":"hogehoge"}
+```
+
+then `created_at` column is set from time attribute in a fluentd packet:
+
+```sql
+mysql> select * from users_mysql_input;
++----+-----------+---------------------+
+| id | user_name | created_at          |
++----+-----------+---------------------+
+|  1 | toyama    | 2016-09-26 18:42:13 |
+|  2 | toyama2   | 2016-09-26 18:42:16 |
+|  3 | toyama3   | 2016-09-26 18:42:19 |
++----+-----------+---------------------+
+3 rows in set (0.00 sec)
+```
+
+## Configuration Example(bulk insert with time format placeholder for table name)
```

Comment:
> But this README.md will see v0.14.X and v0.12.X and v0.10.X users.

Hmm, I see....
I'll add them in descriptions for newly added features.

But I think that we needn't take care of v0.10 users.
Because v0.10 officially announced EOL at the end of this year: https://groups.google.com/forum/#!topic/fluentd/c4f9j9VMWkA